src/components/ui/TutorialWindow.tsx

```
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import styles from '@styles/TutorialWindow.module.css';
+import { useGameStore } from '@store/store';
```

ESLint

[error] 'useGameStore' is defined but never used.

(@typescript-eslint/no-unused-vars)